### DISCUSSION TITLE
any trained ready "korean" model?

### DISCUSSION THREAD

**Discussion by amirbahalegharn** [SubtitleEdit/subtitleedit#6526] [Q&A]
hi
anyone knows any pretrained ready to use Model to use like other VOSK ones ? lots of classic korean movies aren't getting eng subs, so it would be great to make machine subs for them so that viewers can understand what's going on while also see that kudos & credits to subeedit for that
thanks in advance

**Comment by nshmyrev** [comment]
We gonna have Korean model in January. Meanwhile Whisper should transcribe Korean.

**Reply by iPsych** [reply]
@nshmyrev I found this reply just after left the question in the VOSK repo. Will the Korean VOSK model released in January?

**Reply by nshmyrev** [reply]
Yes, we are working on it
